src/applications/personalization/profile/util/contactInformation.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { formatAddress } from 'platform/forms/address/helpers';
+import ReceiveTextMessages from 'platform/user/profile/vap-svc/containers/ReceiveTextMessages';
+import { FIELD_NAMES } from '@@vap-svc/constants';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
+
+const renderContactInformation = (data, type, fieldName) => {
```

Comment:
Is there any reason this can't be a component instead of a function that returns JSX?